.gitignore

```diff
@@ -23,4 +23,7 @@ node_modules
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
-.next
\ No newline at end of file
+.next
+
+*/dist
```

Comment:
It seems dist files still made it into the pull request even tho you updated git ignore.